presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilter.java

```diff
@@ -69,6 +70,11 @@
 
     boolean testBoolean(boolean value);
 
+    default boolean testBoolean(byte value)
```

Comment:
can you please add a javadoc on why this testBoolean method with byte value ? Looking at the surface, I expected this method to be testByte.